crates/smartmodule-development-kit/Cargo.toml

```diff
@@ -13,6 +13,10 @@ name = "smdk"
 path = "src/main.rs"
 doc = false
 
+[features]
+default = ["test-file-io"]
+test-file-io = ["fluvio-cli-common/file-records"]
```

Comment:
is there a reason to make this optional? unless we are building non file-records CLI version.  Just add this feature flag to the dependency.